libs/game/controller.ts

```diff
@@ -122,14 +125,29 @@ namespace controller {
 
         /**
          * Indicates if the button is currently pressed
-        */
+         */
         //% weight=96 blockGap=8 help=controller/button/is-pressed
         //% blockId=keyispressed block="is %button **button** pressed"
         //% group="Single Player"
         isPressed() {
             return this._pressed;
         }
 
+        /**
+         * Indicates how hard the button is pressed, 0-512
+         */
+        //% weight=99 blockGap=8 help=controller/button/pressure-level
+        //% blockId=key_pressure_level block="%button **button** pressure level"
+        //% group="Single Player"
+        pressureLevel() {
```

Comment:
We should discuss before adding more blocks, especially hardware-dependent ones. Can we remove the block definition for now? This is only useful for pygamer, right?

Reply:
removed block; it also works on Adafruit Joy Bonnet, and I assume it will also work with the new sim

but I guess we would actually rather have the block for `player1.analog = true`